aries-core/src/js/components/helpers/Anchors/AnchorCallToAction.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Anchor, ThemeContext } from 'grommet';
+import { FormNext } from 'grommet-icons';
+
+export const AnchorCallToAction = ({ color, ...rest }) => {
+  return (
+    <ThemeContext.Extend
+      value={{
+        anchor: {
+          extend: {
+            'font-weight': '700',
+          },
+          textDecoration: 'none',
+        },
+      }}
+    >
```

Comment:
Why are we duplicating the AnchorUndecorated instead of trying to reuse it?
If it is because the ` 'font-weight': '700',` then maybe it should be included on AnchorUndecorated, especially if both Sidebar and NavHeader will be using it.

Reply:
Sidebar and NavHeader will have font-weights of 400; the green links with arrows are spec'd at 700.

Reply:
I see you removed AnchorUndecorated on your last commit, thanks for the clean-up.
IMHO grommet should allow weight prop so we can avoid this theme context styling, and clean this component out, but it will be a separate effort from this PR.
Regardless, `textDecoration: 'none',` should be moved to the theme instead of being applied on every Anchor/Nav ThemeContext.